### ISSUE TITLE
`shinyFiles` last update (v0.7.0) backward compatibility issue

### ISSUE THREAD

**Issue by sylvainschmitt** [thomasp85/shinyFiles#88]
Hello, I am using `shinyFiles` into my CRAN package `SSDM` for the graphical user interface. A user spotted recently that the change brought in the shinyFiles last version (0.7.0) brought a bug in the user interface (https://github.com/sylvainschmitt/SSDM/issues/32). I'll have a look into it to try to understand what happened. In the meantime do you have a specific list of changes from the version 0.6.0 to 0.7.0 ?

**Comment by sylvainschmitt** [comment]
Ok I was able to solve the issue linked to a change in the default behavior of shinyFiles button. But this means that this new version of `shinyFiles` is not backward compatible. It should be avoided or noticed to reverse imports (CytobankAPIstats, dextergui, SSDM, strataG, xROI).

**Comment by vnijs** [comment]
There were some important changes in this version of `shinyFiles`. However, I ran reverse dependency checks on these packages before `shinyFiles` was submitted to CRAN and found no test failures. 

@sylvainschmitt Would you mind also testing using the  command below? This is the build for PR https://github.com/thomasp85/shinyFiles/pull/86 that is pending evaluation by @thomasp85 

```
devtools::install_github("vnijs/shinyFiles@rstudio-icons")
```

You can install binaries for Windows and Mac using:

install.packages("shinyFiles", repos = "https://radiant-rstats.github.io/minicran/")

**Comment by sylvainschmitt** [comment]
Hello @vnijs , I understand that you needed to do important changes, and that's not an issue I can force users to switch to minimum version 0.7.0 of `shinyFiles` when using `SSDM` with the DESCRIPTION file.

Regarding the reverse dependency, it's partly my fault, because I did not include formal testing of the shiny app in my `SSDM` package. I didn't knew how to do back when I first developed it. That's why I think the reverse dependency test did not throw any warnings. I wonder if it's the same case for other reverse imports. Anyway I should consider including shiny app formal testing in the `SSDM` package to avoid those kind of issue in the future.

I'll try your version below and let you know how it goes.
